mivs/site_sections/mivs_judging.py

```diff
@@ -39,6 +39,8 @@ def game_review(self, session, message='', **params):
                 message = 'You must select a Game Status to tell us whether or not you were able to download and run the game'
             elif review.game_status == c.PLAYABLE and not review.game_score:
                 message = 'You must indicate whether or not you believe the game should be accepted'
+            elif not review.game_status == c.PLAYABLE and review.game_score:
```

Comment:
Style nitpick: can you change ``not review.game_status == c.PLAYABLE`` to ``review.game_status != c.PLAYABLE``?